Ambit/Source/Ambit/Mode/GltfExporterExternal.h

```diff
@@ -0,0 +1,46 @@
+//   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  
+//   Licensed under the Apache License, Version 2.0 (the "License").
+//   You may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//  
+//       http://www.apache.org/licenses/LICENSE-2.0
+//  
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+#pragma once
+
+#include "Exporters/GLTFLevelExporter.h"
+#include "Serialization/BufferArchive.h"
+
+#include "Ambit/Mode/GltfExporterExternalInterface.h"
+
+#include "GltfExporterExternal.generated.h"
+
+
+/**
+ * Class that forwards all Ambit function calls to the external GLTF Exporter plugin.
+ */
+UCLASS()
+class UGltfExporterExternal : public UObject, public IGltfExporterExternalInterface
```

Comment:
Minor: Consider `Wrapper` instead of `External`.

Reply:
Following offline feedback from Kris:
> If it's an interface, though, I wouldn't use the word "wrapper". Wrapping another object is an implementation detail, not an API detail. (Unless, of course, you're defining an IWrapper interface with methods like Wrap() and Unwrap() )

Also, removing the `External` leads to the following issue:
> clashes with what is already defined in the engine by the external plugin (even though it is defined as UGLTFExporter)